html-forms/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form>
+      <div>
+        <label for="email">Email</label>
+        <br>
+        <input type="email" for="email" id="email">
+      </div>
+      <div>
+        <label for="whycode">Why is coding cool?</label>
+        <br>
+        <select name="whycode" id="whycode">
+          <option value="">Select an option</option>
+          <option value="Because it's interesting">Because it's interesting</option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        Favorite Language (choose one)
+        <br>
+        <input type="radio" id="HTML" name="favorite" value="HTML">
+        <label for="favorite">HTML</label>
```

Comment:
The `for` attribute value here should correlate to the item it matches to. In this case, it is the HTML radio input, so the value here should be the `id` on that element, which is `"HTML"`